guacamole/src/main/webapp/app/manage/controllers/manageConnectionGroupController.js

```diff
@@ -105,15 +105,15 @@ angular.module('manage').controller('manageConnectionGroupController', ['$scope'
     permissionService.getPermissions(authenticationService.getCurrentUserID())
             .success(function permissionsReceived(permissions) {
                 
-         // Check if the user has UPDATE permission
-         $scope.hasUpdatePermission =
-               PermissionSet.hasSystemPermission(permissions, PermissionSet.SystemPermissionType.ADMINISTER)
-            || PermissionSet.hasConnectionPermission(permissions, PermissionSet.ObjectPermissionType.UPDATE, identifier);
-            
-         // Check if the user has DELETE permission
-         $scope.hasDeletePermission =
-               PermissionSet.hasSystemPermission(permissions, PermissionSet.SystemPermissionType.ADMINISTER)
-            || PermissionSet.hasConnectionPermission(permissions, PermissionSet.ObjectPermissionType.DELETE, identifier);
+        // Check if the connection group is new or if the user has UPDATE permission
+        $scope.canSaveConnectionGroup = !identifier || 
```

Comment:
When there is no identifier, and thus a user is creating a connection group on save, the user can save if and only if they have connection group creation permission... or is a sysadmin.